test/form-helper.test.js

```diff
@@ -155,6 +155,94 @@ describe('form-helper functions', () => {
 
             expect(actual).toMatch(expected);
         });
+
+        it('Should show sign-in hyperlink by default', () => {
```

Comment:
Due to the [default value in renderSection](https://github.com/CriminalInjuriesCompensationAuthority/cica-web/blob/141c70049f02222599fb813aa0710911d8fd4634/questionnaire/form-helper.js#L48) the [shouldShowHyperlink](https://github.com/CriminalInjuriesCompensationAuthority/cica-web/blob/141c70049f02222599fb813aa0710911d8fd4634/questionnaire/form-helper.js#L218) function isn't being tested